api/src/models/estimate.schema.ts

```diff
@@ -0,0 +1,10 @@
+import * as mongoose from 'mongoose';
```

Comment:
remove this and other schemas from this PR and create a separate PR for the rest of schemas.